src/main/java/org/websoso/WSSServer/domain/UserNovel.java

```diff
@@ -54,6 +60,12 @@ public class UserNovel extends BaseEntity {
     @JoinColumn(name = "novel_id", nullable = false)
     private Novel novel;
 
+    @OneToMany(mappedBy = "userNovel", cascade = ALL, fetch = FetchType.LAZY)
+    private List<UserNovelAttractivePoint> userNovelAttractivePoints = new ArrayList<>();
+
+    @OneToMany(mappedBy = "userNovel", cascade = ALL, fetch = FetchType.LAZY)
+    private List<UserNovelKeyword> userNovelKeywords = new ArrayList<>();
+
```

Comment:
p2;
부모 엔티티로 자식 엔티티 생명주기를 관리하려고 한다면 orphanRemoval도 걸어주면 어떨까욥?
```suggestion
    @OneToMany(mappedBy = "userNovel", cascade = ALL, fetch = FetchType.LAZY, orphanRemoval = true)
```

Reply:
너무너무 좋은 생각입니다! 저번에 얘기가 나왔던 것 같은데 제가 잊었네요😅 추가하겠습니다~!